embrace-android-sdk/src/integrationTest/kotlin/io/embrace/android/embracesdk/testcases/ConfigServiceTest.kt

```diff
@@ -0,0 +1,67 @@
+@file:Suppress("DEPRECATION")
+
+package io.embrace.android.embracesdk.testcases
+
+import android.os.Build.VERSION_CODES.TIRAMISU
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import io.embrace.android.embracesdk.Embrace.AppFramework
+import io.embrace.android.embracesdk.IntegrationTestRule
+import io.embrace.android.embracesdk.recordSession
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertTrue
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.annotation.Config
+
+/**
+ * Validation of the basic and miscellaneous functionality of the Android SDK
+ */
+@Config(sdk = [TIRAMISU])
+@RunWith(AndroidJUnit4::class)
+internal class ConfigServiceTest {
```

Comment:
Ideally we should also have a test case that checks the SDK can start again after it's been stopped (i.e. that it's capable of fetching remote config even when disabled).